frontend-ssr/pages/api/notifications/mock.ts

```diff
@@ -0,0 +1,57 @@
+import { uuid } from '~/services/Utils';
+import { Notification } from '../../../redux/user/types';
+
+const MOCK_NOTIFICATIONS: Notification[] = [
```

Comment:
Good job with the mocked info, but let's not put it in the `api` folder. Move it in `UserService.ts` since it's the only place we use it. So that we only modify thing in one file when needed.